tests/e2e_test.go

```diff
@@ -51,6 +51,10 @@ var _ = ginkgo.BeforeSuite(func() {
 var _ = ginkgo.AfterSuite(localUtils.TearDownNetwork)
 
 var _ = ginkgo.Describe("[Teleporter integration tests]", func() {
+	// Cross-chain application tests
```

Comment:
after the comment below, and we issue tx to propose vm to all subnets, I'd like for validator churn to not need to be added last. Do you know right now if these tests are in random order?

Reply:
Agreed. If we can fix it we should. Otherwise what happens if we have another test that also needs to go last? 🙃

Reply:
instead of making the issue to proposevm calls to each subnet individually, can we iterate through all subnets in the network, get subnet info, and issue tx? To prevent in the future if we add more subnets for this test to break and need to be debugged.